shardingsphere-features/shardingsphere-readwrite-splitting/shardingsphere-readwrite-splitting-core/src/main/java/org/apache/shardingsphere/readwritesplitting/algorithm/WeightReplicaLoadBalanceAlgorithm.java

```diff
@@ -0,0 +1,99 @@
+package org.apache.shardingsphere.readwritesplitting.algorithm;
```

Comment:
Licence missing.

Reply:
I’m very sorry,I don’t know how to deal with the license missing.

Reply:
Just copy from another file, then commit and push again.